prow/monitoring/prow_prometheusrule.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: prow
+    role: alert-rules
+  name: prow
+  namespace: prow-monitoring
+spec:
+  groups:
+  - name: ci-absent
+    rules:
+    - alert: deckDown
+      annotations:
+        message: The service deck has been down for 5 minutes.
+      expr: |
+        absent(up{job="deck"} == 1)
+      for: 5m
+      labels:
+        severity: slack
+    - alert: ghproxyDown
+      annotations:
+        message: The service ghproxy has been down for 5 minutes.
+      expr: |
+        absent(up{job="ghproxy"} == 1)
+      for: 5m
+      labels:
+        severity: slack
+    - alert: hookDown
+      annotations:
+        message: The service hook has been down for 5 minutes.
+      expr: |
+        absent(up{job="hook"} == 1)
+      for: 5m
+      labels:
+        severity: slack
+    - alert: jenkins-dev-operatorDown
```

Comment:
This is specific to us -- look at the services defined in `test-infra/prow/cluster/` to see what to define here

Reply:
Leave only deck there. This file will be generated by `mixin`.
For now, it is just to prove that components in stack work together.